src/web/app/pages-instructor/instructor-home-page/instructor-home-page.component.ts

```diff
@@ -136,7 +141,7 @@ export class InstructorHomePageComponent extends InstructorSessionBasePageCompon
   loadCourses(): void {
     this.courseTabModels = [];
     this.httpRequestService.get('/courses').subscribe((courses: Courses) => {
```

Comment:
Change this to use course.service.ts instead of making a direct call?

Reply:
I have seen this in your implementation, but I'm not sure why we should do this?
Note that one of consolidation requirement is: 
> Change frontend API request only if necessary, we will do the consolidation on top of the existing APIs. It would a bit messy to change the frontend simultaneously (as there are a lot of RPC calls in the old endpoints).

Reply:
Fair. I was under the impression that this was necessary, which was why I did it. Rahul has clarified with me that it isn't.